README.md

```diff
@@ -0,0 +1,52 @@
+[![Build Status](https://travis-ci.com/oasis-roles/openstack_undercloud.svg?branch=master)](https://travis-ci.com/oasis-roles/openstack_undercloud)
+
+openstack_undercloud
+===========
+
+Basic description for openstack_undercloud
+
+Requirements
+------------
+
+Ansible 2.4 or higher
+
+Red Hat Enterprise Linux 7 or equivalent
+
+Valid Red Hat Subscriptions
+
+Role Variables
+--------------
+
+Currently the following variables are supported:
+
+### General
```

Comment:
Other vars to be document later, I presume.

Reply:
Indeed